draft-claise-semver.md

```diff
@@ -87,24 +86,23 @@ semver.org):
 * BETA is incremented when a new pre-release or testing version is made. 
 
 
-In IETF terms, this versioning scheme provides functionality equivalent to
+In IETF terms, this versioning scheme provides functionality analagous to
 several parts of the traditional IETF process.
 
-* MAJOR replaces the "obsoletes" relation between RFCs
-* MINOR replaces the "updates" relation between RFCs
```

Comment:
Both "updates" and the errata process get used for all kinds of things, and the semantics of what is being proposed here are only a subset of each of those. There has been a lot of debate recently about "updates" in particular, so we should be clear that we're analogizing to just one form of "update."